### PR TITLE
This is Mina's password login program

### PR DESCRIPTION
**Pull request by awadmina**
**Why**

I wanted to try to fork in github
---

**What**

password file and README file
---

**Verification**

run "python3 password/password.py" in terminal to run the program. make sure you have python3 installed
---

**Checklist:**

- [x] This contribution abides by all rules listed in `docs/Rules.md`

### CODE REVIEW

File: submissions/password/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This is a simple demonstration of how the string works in python.
+The demonstration lets to create a password and then asks you for the password when it loads.
+
+
+if you want to run in terminal type "python3 password/password.py"
```
Comment (SoorajModi) [inline submissions/password/README.md:5]: Missing the following in the README:
- List of contributors
- Why you contributed it

**Comment by SoorajModi** [comment]
Take a look at #1 for an example, improve the name of the PR, also the checkbox should look like `[x]` not `[ x]`

**Review by SoorajModi — CHANGES_REQUESTED** [review]

File: submissions/password/README.md
Comment (MarshallAsch) [inline submissions/password/README.md:1]: Try to use markdown formatting to make the different sections easy to read. See [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) for some more info on markdown syntax

File: submissions/password/password.py
Comment (MarshallAsch) [inline submissions/password/password.py:1]: SPACING!!!!! Leave some whitespace in the code to visionally separate the code into meaningful blocks. As a single large wall of text its a bit difficult to tell whats happening and what each component does.

File: submissions/password/password.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import sys
+import time
+def slowprint(s):
+  for c in s + '\n':
+    sys.stdout.write(c)
+    sys.stdout.flush()
+    time.sleep(2./10)
+if __name__ == "__main__":
+ print("what do you what your username to be?")
```
Comment (MarshallAsch) [inline submissions/password/password.py:9]: Does this work? it looks like your indentation is inconsistent in the file?

**Review by MarshallAsch — CHANGES_REQUESTED** [review]

**Comment by SoorajModi** [comment]
Hey @awadmina there are a few outstanding issues left to resolve for this PR, please address them ASAP so we can merge in your contribution. To make changes you can push to your branch, and the changes should appear on this PR.

**Pull request closed without merging**